bin/.travis/prepare_behat.sh

```diff
@@ -6,6 +6,11 @@
 git fetch --unshallow && git checkout -b tmp_travis_branch
 export BRANCH_BUILD_DIR=$TRAVIS_BUILD_DIR
 export TRAVIS_BUILD_DIR="$HOME/build/ezplatform"
+
+## Get branch alias from composer.json and if not set fallback to using TRAVIS_BRANCH
+ba="\$a['extra']['branch-alias']"
+branch_alias=$(php -r "\$a=json_decode(file_get_contents('composer.json'),true);print isset($ba) ? $ba[key($ba)]: $TRAVIS_BRANCH;")
```

Comment:
yep :)

was trying to not end up with to long line, but it indeed just makes it even harder to read so I can revert that, but see comment on PR in regards to this being far from perfect and hardcoded alias is safest no matter what..